FLOE-494: Upgraded to Infusion 2.0

jobara opened the pull request:
- Upgraded to Infusion 2.0
- Fixed a styling issue that prevented UIO from being visible with the Twenty Seventeen theme
- Restructured files by adding a src directory

https://issues.fluidproject.org/browse/FLOE-494

jobara committed:
FLOE-494: Upgrading to Infusion 2.0

jobara committed:
FLOE-494: Updating plugin to work with Infusion 2.0

Also added uio.css to fix styling issues with Wordpress' twentyseventeen theme.

jobara committed:
FLOE-494: Restructured the repository to include a src directory

jobara committed:
FLOE-494: Fixing issue where the wrong files were modified.

set-defaults.php and inject-dependencies.php were backwards.

cindyli commented on readme.txt:
```diff
@@ -39,7 +39,13 @@ for tips on styling your site to work nicely with UIO.
 = Are there any themes themes that already work well with UIO? =
```
double wording on `themes`. Please remove one. Thanks.

cindyli commented on readme.txt:
```diff
@@ -39,7 +39,13 @@ for tips on styling your site to work nicely with UIO.
 
-http://issues.fluidproject.org/browse/FLOE/component/11510#selectedTab=com.atlassian.jira.plugin.system.project%3Acomponent-issues-panel
\ No newline at end of file
+http://issues.fluidproject.org/browse/FLOE/component/11510#selectedTab=com.atlassian.jira.plugin.system.project%3Acomponent-issues-panel
```
This link doesn't work. Also, the link at line 36 points at an UIO doc on the wiki.fluidproject.org site. Shall it be updated to point to the infusion doc site?

jobara replied:
The one at line 36 is only a placeholder in the Infusion Docs site, but this will future proof the readme a bit, so I can change it.

cindyli commented on src/php/inject-dependencies.php:
```diff
@@ -0,0 +1,36 @@
+<?php
+/*
+Copyright 2015 OCAD University
```
Fix the copyright. The same comment applies to other files.

jobara replied:
I didn't really change the contents of most of the files, so I left the copyrights. For the most part I just moved them.

jobara committed:
FLOE-494: Updated readme.txt based on review comments.

jobara commented:
@cindyli updated and ready for more review.

jobara committed:
FLOE-494: Moved source files back to root.

This is done to facilitate Wordpress' EDIT feature, which allows a WP admin to edit a plugin.
It could only access files at the root.

jobara commented:
@cindyli ready for more review and updated the [dev guide](https://wiki.fluidproject.org/display/fluid/UI+Options+Wordpres+Plugin+Development+Guide)

cindyli commented:
Merged at 2967766f6896de5169e1e5a2b9c4ac08cde6563a
